drivers/serial/Kconfig.stm32

```diff
@@ -20,3 +20,17 @@ config UART_STM32
 	  This option enables the UART driver for STM32 family of
 	  processors.
 	  Say y if you wish to use serial port on STM32 MCU.
+
+config UART_STM32_UNALIGNED_DMA
+	bool "STM32 MCU UART Unaligned DMA Support"
+	depends on UART_ASYNC_API
+	depends on CPU_CORTEX_M7
+	default y
+	help
+	  Support unaligned DMA buffer. When DMA buffers are not aligned on
+	  cache line boundaries, or there size is not a multiple of the
```

Comment:
s/there/their/

Reply:
@dcpleung I do not catch your point, sorry. This text is coming from the https://github.com/zephyrproject-rtos/zephyr/pull/27911/commits/70e111c1ae19e3aabfa1a95e906d40021542444b